build.gradle

```diff
@@ -7,6 +7,16 @@ buildscript {
     }
 }
 
+plugins {
+    id "com.gradle.build-scan" version "1.12.1"
+}
+
+buildScan {
+    termsOfServiceUrl = 'https://gradle.com/terms-of-service'
+    termsOfServiceAgree = 'yes'
+    publishAlwaysIf(System.getenv('CI') != null)
```

Comment:
Do we ever set this variable anywhere?

Reply:
This is set in Travis env [by default][1]

[1]: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

Reply:
Oh I didn’t know this